### ISSUE TITLE
'mock sparsity' 

### ISSUE THREAD

**Issue by tatami-galaxy** [google-research/jaxpruner#4]
I'm new to sparsity. So I'm a little confused by this sentence in the readme : 

"We mock sparsity by using binary masks and use dense operations for simulating sparsity. In the longer run, we also plan to provide integration with the [jax.experimental.sparse](https://jax.readthedocs.io/en/latest/jax.experimental.sparse.html), aim to reduce the memory footprint of our models."

What is meant by 'we mock sparsity'?

**Comment by evcu** [comment]
It means the sparse parameters are represented as dense matrices together with binary masks of matching shape. I would recommend checking the quick start colab, where you can find these parameters and masks.
